webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue

```diff
@@ -204,6 +214,13 @@ export default {
       } else {
         setTimeout(this.hideDocPreviewComments, intervalCheck);
       }
+    },
+    openHome() {
```

Comment:
```suggestion
    goBack() {
```
because we do not always redirect to the home page.

Reply:
yes ok